src/components/QueryEditor.tsx

```diff
@@ -165,7 +165,7 @@ export function QueryEditor(props: Props) {
         <SQLEditor
           query={query}
           onRunQuery={props.onRunQuery}
-          onChange={(rawQuery) => props.onChange({ ...props.query, rawQuery })}
+          onChange={(query) => props.onChange(query)}
```

Comment:
could this be simplified to this?
```suggestion
          onChange={props.onChange}
```

Reply:
Oh yeah, missed that one, thanks!